cmd/kubeadm/app/apis/kubeadm/bootstraptokenhelpers.go

```diff
@@ -0,0 +1,154 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubeadm
```

Comment:
I'm wondering if those helpers should be here - as part of the kubeadm specific API -  or into `k8s.io/client-go/tools/bootstrap/token/` - as part of the common infrastructure for boostrap tokens -. WDYT?

Reply:
They will move when the types do.